### PR TITLE
Pods deployment status display in OLM UI

### LINKED ISSUE #53: Use doughnuts to visualize health of cluster members
The etcd operator has an interesting example of how to visualize overall health of the cluster members using this doughnuts. I also learned this trick this week and the documentation for it is hard to find.  Details here:

https://github.com/openshift/console/tree/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors

https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/types.ts

### PR DESCRIPTION
**Pull request by dmvolod**
This fix able to display Pods deployment status in the OLM UI as well as able to change `replicas` field in UI
image

**Able to change replicas field in UI**
image

fixes #53 #408

### CODE REVIEW

File: deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml
Comment (oraNod) [inline deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml:94]:
```suggestion
          - description: The desired number of nodes for the Infinispan cluster.
```

File: deploy/crds/infinispan.org_infinispans_crd.yaml
Comment (oraNod) [inline deploy/crds/infinispan.org_infinispans_crd.yaml:826]: @dmvolod Does this mean "not able to start" due to some fault? When would this description appear to users?

File: deploy/crds/infinispan.org_infinispans_crd.yaml
Comment (oraNod) [inline deploy/crds/infinispan.org_infinispans_crd.yaml:821]:
```suggestion
                  description: Deployments are starting
```

File: deploy/crds/infinispan.org_infinispans_crd.yaml
Comment (oraNod) [inline deploy/crds/infinispan.org_infinispans_crd.yaml:816]:
```suggestion
                  description: Deployments are ready to handle client requests
```

File: deploy/crds/infinispan.org_infinispans_crd.yaml
Comment (dmvolod) [reply deploy/crds/infinispan.org_infinispans_crd.yaml:826]: @oraNod this is auto-generated values from the RedHat syseng operator-utils component. We can't to update values in our code

File: deploy/crds/infinispan.org_infinispans_crd.yaml
Comment (dmvolod) [reply deploy/crds/infinispan.org_infinispans_crd.yaml:821]: @oraNod this is auto-generated values from the RedHat syseng operator-utils component. We can't to update values in our code

File: deploy/crds/infinispan.org_infinispans_crd.yaml
Comment (dmvolod) [reply deploy/crds/infinispan.org_infinispans_crd.yaml:816]: @oraNod this is auto-generated values from the RedHat syseng operator-utils component. We can't to update values in our code

**Comment by dmvolod** [comment]
@ryanemerson , @rigazilla we still have a floating error on backup/restore test like
`panic: Unexpected response code 404 [recovered]`
I don't have an idea why it happens, probably some function calls early than needed or not wait for some completion

**Review by ryanemerson — COMMENTED** [review]
I don't really know the OLM side of things, but the controller changes LGTM. Can you remove 205f71c before merging

**Comment by dmvolod** [comment]
> I don't really know the OLM side of things, but the controller changes LGTM. Can you remove [205f71c](https://github.com/infinispan/infinispan-operator/commit/205f71c5852d80d4d9432941f5e82eeffeb99f26) before merging

Yes, sure. We don't need this commit anymore. But I would like to wait for #707 to be merged for full test.

**Comment by dmvolod** [comment]
> I don't really know the OLM side of things, but the controller changes LGTM. Can you remove [205f71c](https://github.com/infinispan/infinispan-operator/commit/205f71c5852d80d4d9432941f5e82eeffeb99f26) before merging

@ryanemerson please pay attention, that #770 didn't solve `404` error and brings another kind of floating error with sporadic number of restored records

**Review by rigazilla — APPROVED** [review]
LGTM!

**Comment by ryanemerson** [comment]
Thanks @dmvolod

**Pull request merged**
